src/js/DataTables/DataTable.js

```diff
@@ -213,6 +213,17 @@ export default class DataTable extends PureComponent {
     delete props.defaultSelectedRows;
     delete props.baseId;
     delete props.onRowToggle;
+    const {
+      width,
+      height,
+      ...style
+    } = props.style || {};
+    delete props.style;
+    if (!responsive) {
+      style.width = width;
+      style.height = height;
+    }
+    props.style = style;
```

Comment:
I think instead of extracting the `width` and `height` from the `style` prop, I think it might be better to go with something like:

```
  fixed: PropTypes.shape({
    width: PropTypes.number,
    height: PropTypes.number.isRequired,
  }),

  /* render() */
  const { height, width } = this.props.fixed;
```

My reasoning is that I would like `style` to be consistent -- it will always apply all styles to the same thing.

It also is a bit more self-documenting specifying that you would like to have a fixed table.